sdks/rust/src/worker/mod.rs

```diff
@@ -16,6 +16,7 @@
  * limitations under the License.
  */
 
+pub mod data;
```

Comment:
[nits] better to be `pub(in crate::worker)`?

Reply:
Agreed, but that also meant moving data_test into the crate to be able to test the channel implementation.
Like I mentioned, I'll likely rework these tests at some point to use turmoil and once the SDK is fleshed out a bit more this can move back to tests with all the other integration tests, since we'll be able to construct fake pipelines that communicate with a fake FnAPI server implementation when this is closer to complete. I think the visibility of the module and documenting the expected behavior is more important to get right at this stage than it is to have integration tests exactly where they need to be, so that's fixed now. :)